src/runtime/meter-toast-api.js

```diff
@@ -117,11 +120,25 @@ export class MeterToastApi {
           this.sendCloseRequestFunction_
         );
         this.win_.addEventListener('mousedown', this.sendCloseRequestFunction_);
-        this.win_.addEventListener('wheel', this.sendCloseRequestFunction_);
         // Making body's overflow property 'hidden' to prevent scrolling
-        // while swiping on the iframe.
-        const $body = this.win_.document.body;
-        setStyle($body, 'overflow', 'hidden');
+        // while swiping on the iframe only on mobile.
+        if (this.isMobile_()) {
+          const $body = this.win_.document.body;
+          setStyle($body, 'overflow', 'hidden');
+        } else {
+          let start, scrollTimeout;
+          this.scrollEventListener_ = () => {
+            start = start || this.win_./*REVIEW*/pageYOffset;
```

Comment:
@ChrisAntaki I see that pageYOffset and scrollY are both banned but don't know of an alternative. Do you think this is okay?

https://github.com/subscriptions-project/swg-js/blob/7bd1026dbb17330c21ac108591eec804a064dd3c/build-system/tasks/check-rules.js#L137

Reply:
Seems fine to me